tests/components/yeelight/test_light.py

```diff
@@ -468,36 +486,52 @@ async def _async_test(
             "rgb_color": bg_rgb_color,
             "xy_color": bg_xy_color,
         },
-        entity_name="ambient ambilight",
-        entity_id="light.ambient_ambilight",
+        name=f"{NAME} ambilight",
+        entity_id=f"{ENTITY_LIGHT}_ambilight",
     )
 
 
 async def test_effects(hass: HomeAssistant):
     """Test effects."""
-    yaml_configuration = {
-        DOMAIN: {
-            CONF_DEVICES: YAML_CONFIGURATION[DOMAIN][CONF_DEVICES],
-            CONF_CUSTOM_EFFECTS: [
-                {
-                    CONF_NAME: "mock_effect",
-                    CONF_FLOW_PARAMS: {
-                        ATTR_COUNT: 3,
-                        ATTR_TRANSITIONS: [
-                            {YEELIGHT_HSV_TRANSACTION: [300, 50, 500, 50]},
-                            {YEELIGHT_RGB_TRANSITION: [100, 100, 100, 300, 30]},
-                            {YEELIGHT_TEMPERATURE_TRANSACTION: [3000, 200, 20]},
-                            {YEELIGHT_SLEEP_TRANSACTION: [800]},
-                        ],
+    assert await async_setup_component(
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6090606)